pkg/proxy/ipvs/proxier.go

```diff
@@ -578,33 +578,34 @@ func (handle *LinuxKernelHandler) GetModules() ([]string, error) {
 	}
 	ipvsModules := utilipvs.GetRequiredIPVSModules(kernelVersion)
 
+	var bmods []string
+
+	// Find out loaded kernel modules. If this is a full static kernel it will thrown an error
+	modulesFile, err := os.Open("/proc/modules")
```

Comment:
Prefer to keep this check in the original location (line 593), close to where the output (`modulesFile`) is consumed.

Reply:
@andrewsykim actually I consume the err here to check if this is not a static kernel. That's why I've changed the location

Reply:
Gotcha, can we move 
```
	mods, err := getFirstColumn(modulesFile)
	if err != nil {
		return nil, fmt.Errorf("failed to find loaded kernel modules: %v", err)
	}
```

up here as well then?

Reply:
For sure! Just did it